hatchways/team-pine
# Implemented pagination on reviews.

### What this PR does:
- Implements pagination for reviews. Reviews are retrieved from the API based on page (10 at a time) and displayed on the modal.

### Screenshots / Videos:
![image](https://user-images.githubusercontent.com/34494229/153539346-a82e8d44-5ccd-434e-bcb3-a30d23c01375.png)
![image](https://user-images.githubusercontent.com/34494229/153539379-af8d0bb6-e9a5-41f5-814a-e58aace6f90f.png)
![image](https://user-images.githubusercontent.com/34494229/153539399-ead3b0e3-5725-476c-ab11-b99cd6cc5598.png)
![image](https://user-images.githubusercontent.com/34494229/153539438-57535df4-a785-41b9-8dec-eaf1e5482a30.png)


### Any information needed to test this feature:
- You need 11 non sitter accounts and 1 sitter account to test this feature (21 non sitters to test skipping pages).
- Navigate to the sitter profile via profile listings (if integrated) or directly through /profile/{profile id}.
- Create a review
- Repeat the previous two steps for all non sitter accounts
- Three reviews maximum should display on the main part of the page. Click See All Reviews to see the modal.
- Click on the pages to test reviews properly switching. Additionally, if you click back to a visited page within five seconds, you should not notice an additional network request. Skipping pages should perform in the exact same way.

### Any issues with the current functionality:
- There is no sorting functionality.
- Main doesn't currently have availability properly structured into profile settings, so profile listings couldn't be paginated.

closes #14
